CONTRIBUTING.rst

```diff
@@ -0,0 +1,15 @@
+Thanks for considering submitting a pull request, we really appreciate that!
+Before doing so, here are a few guidelines:
+
+* You agree to license your contributions under the
+  [license](https://github.com/mozilla/kinto/blob/master/LICENSE).
+* Use pull-requests early so it's open for discussion, even if your
+  contribution isn't ready yet.
+* All pull requests should include tests, they help us avoiding regressions in
+  our code.
+* A pull-request adding functionality should also include docs.
```

Comment:
ah, sorry, I thought you meant I should update the contributing.html documentation file ;) Thanks for the rewording advice, will do that straight away